packages/react-strict-dom/src/native/modules/createStrictDOMComponent.js

```diff
@@ -153,6 +153,54 @@ function resolveTransitionProperty(property: mixed): string[] {
   return [];
 }
 
+//custom radio button component 
+const RadioButton = ({ options, selectedOption, onSelect }) => {
```

Comment:
This does not work like `<input type="radio">` on the web.

There is no "options" attribute. We would need to automatically unselect other radio inputs when another with same name is selected.

We would need to integrate with the nearest form.

Etc.